lib/config/presets/index.ts

```diff
@@ -168,6 +168,11 @@ export function parsePreset(input: string): ParsedPreset {
   ) {
     presetSource = 'internal';
     [repo, presetName] = str.split(':');
+  } else if (str.startsWith('regexManagers:')) {
```

Comment:
I'm certain I recall seeing preset migrations elsewhere, but this was a while ago... For some reason today I couldn't find them. 

Appreciate feedback if there is a better spot for this

Reply:
ty @viceice - moved to there